neuralmonkey/runners/rnn_runner.py

```diff
@@ -241,19 +241,19 @@ def next_to_execute(self) -> NextExecute:
             raise Exception(
                 "Nothing to execute, if there is already a result.")
 
-        to_run = {'logprobs': self._decoder.train_logprobs[self._time_step]}
+        to_run = {'logprobs': self._decoder.runtime_logprobs[self._time_step]}
 
         self._current_beam_batch = self._to_exapand.pop()
 
         if self._current_beam_batch is not None:
             additional_feed_dict = {t: index for t, index in zip(
-                self._decoder.train_inputs[1:],
+                tf.unpack(self._decoder.train_inputs[1:]),
```

Comment:
Tak co se tady má feedovat? Nic? Nebo nějakej tenzor? RNN Runner by mohl z dekodéru používat jen tu funkci decoding_step. Ale do přepisování RNN runneru se hrnout nechci,  protože tomu kódu moc nerozumim. Ale chtělo by to dát sem něco uspokojivějšího, to jo..